src/Navigation/SidePanel/common.tsx

```diff
@@ -17,7 +17,7 @@ export const Button = styled('div')`
   transition: all 0.5s ease-out-in;
   top: 10%;
   left: ${({ style: { width } }: any) => width || 300}px;
-  animation: indicate 1s infinite;
+  animation: indicate 1s 10;
```

Comment:
I'd suggest just keeping it simple and removing the animation. t's graphically unpleasent

Reply:
Are you sure ? the obvious implication is a big diff churn ;)

Reply:
yes I'm sure

Reply:
Done. ( i think this is now ready to be merged )